### ISSUE TITLE
Please Add White Labeling as Higher Option Plan?

### ISSUE THREAD

**Issue by marynol** [pojome/elementor#4567]
Pretty safe to say anyone with an Agency want white labeling. 

Who is with me?

**Comment by themedleb** [comment]
+1

**Comment by themedleb** [comment]
On the back-end and most importantly on the front-end (HTML output), right now I can easily tell if a website/page is made using Elementor only by using the browser inspector.

**Comment by bainternet** [comment]
@marynol 

We do have plans in this direction.

**Comment by marynol** [comment]
lol I was trolled in the FB group for suggesting this a feature a lot of people would love. 

Great to hear Brainternet much appreciated...

**Comment by parfilov** [comment]
+1

**Comment by daconix** [comment]
+1

**Comment by Tim2468** [comment]
+1

**Comment by IamSiddharth** [comment]
+1

**Comment by zrk4444** [comment]
+1

**Comment by alriksson** [comment]
+1  As @themedleb says not limited to the backend, we want a white label on the frontend, a pure white level solution, remove all trace of elementor frontend when it comes to  id, classes, etc etc

**Comment by mamito** [comment]
+1

**Comment by Jakoli** [comment]
> @marynol
> 
> We do have plans in this direction.

Any news? Thank you!

**Comment by ehong33234** [comment]
Any ETA on this?

**Comment by d19dotca** [comment]
I think the recent(ish) price hikes for the new Studio & Agency packages should definitely make this feature request even more relevant, as Elementor seems to be prioritizing larger customers or at least recognizing many large customers of Elementor who are in a position where White Labelling would be more applicable.

**Comment by cranstantinople** [comment]
Yes, please.  Elementor has eliminated the need for Dozens of plugins for me already and increased stability for my sites.  White Label with the ability for custom Dashboard/Admin menu would be very welcomed.
